schemas/json/theme.json

```diff
@@ -796,7 +797,28 @@
 					"$ref": "#/definitions/settingsPropertiesComplete"
 				},
 				"core/image": {
-					"$ref": "#/definitions/settingsPropertiesComplete"
+					"type": "object",
+					"allOf": [
+						{
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						{
+							"type": "object",
+							"properties": {
+								"behaviors": {
+									"description": "Allows users to set named behaviors for a block.\nGutenberg plugin required.",
+									"type": "object",
+									"properties": {
+										"lightbox": {
+											"description": "Show a lightbox for the image.\nGutenberg plugin required.",
```

Comment:
The wording is inconsistent in the entire `theme.json` schema, so it's difficult to understand the intent for all settings. However, I believe it's all about deciding whether a UI control should be available to users.

Reply:
> so the lightbox might still be enabled - it's just the user can't control it

Oh, I think we didn't know this bit.

cc: @santosguillamot

Reply:
Me neither! Thanks for sharing 🙂